packages/pn-commons/src/components/FileUpload.tsx

```diff
@@ -140,6 +139,12 @@ const FilenameBox = ({ filename }: { filename: string }) => {
   );
 };
 
+const TextFieldDisabled = styled(TextField)(({ theme }) => ({
```

Comment:
Why we need this? Generally we avoid to customize mui component

Reply:
I want to change color of disabled TextField. It's not an editable field but in mockup it's designed with style as an input.

Reply:
Ok I understood. We can use readonly property instead of disabled, and avoid customization. What do you think?

Reply:
Yes, great!